examples/discovery-search-app/cypress/integration/basic/spec.ts

```diff
@@ -21,18 +21,18 @@ describe('Basic search', () => {
   // Rendering initial page
   describe('When the example app loads', () => {
     it('SearchInput has placeholder text "Search"', () => {
-      cy.get('.bx--search-input').should('have.attr', 'placeholder', 'Search');
+      cy.findByPlaceholderText('Search').should('have.attr', 'placeholder', 'Search');
```

Comment:
Since it's finding by the placeholder text now, isn't the `should` check essentially checking for the same thing it's found by? In this case, I think it could be clearer to just test for should exist.